Sources/Commands/SwiftTool.swift

```diff
@@ -46,6 +48,10 @@ public class SwiftTool<Mode: Argument, OptionType: Options> {
         let dynamicSelf = type(of: self)
         do {
             (self.mode, self.options) = try dynamicSelf.parse(commandLineArguments: args)
+            // Honor chdir option is provided.
+            if let dir = options.chdir {
+                try chdir(dir.asString)
```

Comment:
Should we perhaps have an overload for `AbsolutePath` here?  People often have to use `asString` with `chdir()`.  Or perhaps a call to `localFileSystem` to change the directory would be more appropriate.